packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -168,15 +168,15 @@ describe('AddresslistVoting', function () {
       await expect(
         voting
           .connect(signers[1])
-          .createProposal(dummyMetadata, [], 0, 0, VoteOption.None, false)
+          .createProposal(dummyMetadata, [], 0, 0, 0, VoteOption.None, false)
```

Comment:
No new tests to check how the failure bitmap works? Separate task?

Reply:
https://github.com/aragon/core/pull/207/files#diff-2a15c769379f924880f7d57bdc744281b5db84e22fcf521247a06747a7e5966fR285 Check this. The same thing is in every plugin.

Reply:
Right, but I mean something like `0b0000101` and then, checking that actions 1 and 3 can fail without execution actually reverting.

Same with actions 2, 4, 5, 6, 7, 8 failing, provoking the whole execution to fail.

We can create a separate task to add this later, but it has to exist

Reply:
I don't think thats the test that should be here . I already wrote them in `dao.ts` ! for these tests, you only care about if event was emitted with a correct failureMap.  Otherwise, you really are testing this with kind of e2e or integration.

feel free to create a task if you still want.